codalab/model/orm_object.py

```diff
@@ -38,6 +38,7 @@ class ORMObject(object):
         Tuple[str, str, str, str, str, str],
         Tuple[str, str, str, str],
         Tuple[str, str, str, str, str, str, str],
+        Tuple[str, str, str, str, str, str, str, str],
```

Comment:
Could we just replace this entire thing with `Tuple[str, ...]` which works for all variable-length tuples (see https://github.com/python/mypy/issues/184#issuecomment-70385687)? The current way we type this property doesn't seem to be really readable / useful and just makes it harder to update columns.

Reply:
Thanks for the suggestion. I updated the type.